### PR TITLE
Update dev docs EVM intro

### PR DESCRIPTION
**Pull request by cradle0fFilth**
From codes of geth we can know that evm is not a single entity. also from ethereumBook in chapter13:“When a transaction results in smart contract code execution, an EVM is instantiated with all the information required in relation to the current block being created and the specific transaction being processed. In particular”，"with each call resulting in another EVM being instantiated around the new target of the call. ",you can find it in 'https://github.com/ethereumbook/ethereumbook/blob/develop/13evm.asciidoc'

## Description

Reconceptualize the EVM with precision, focusing solely on its core attributes without delving into other aspects

### CODE REVIEW

**Review by wackerow — COMMENTED** [review]
Hey @cradle0fFilth, appreciate your suggestion and apologies for the delays getting back to you. I see where you're coming from here but I'm not sure this phrasing here is going to help users understand this better.

Hey @jmcook1186, curious if you have any thoughts on this.. if you think it'd be helpful, or any way to improve it a bit?

**Comment by minimalsm** [comment]
@jmcook1186 any thoughts here?

File: public/content/developers/docs/evm/index.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ description: An introduction to the Ethereum virtual machine and how it relates
 lang: en
 ---
 
-The EVM’s physical instantiation can’t be described in the same way that one might point to a cloud or an ocean wave, but it does _exist_ as one single entity maintained by thousands of connected computers running an Ethereum client.
+The Ethereum Virtual Machine(EVM) is the runtime environment for smart contracts in Ethereum.Every time a transaction is executed, a new EVM instance is created to maintain a unique world state machine.
 
 The Ethereum protocol itself exists solely for the purpose of keeping the continuous, uninterrupted, and immutable operation of this special state machine. It's the environment in which all Ethereum accounts and smart contracts live. At any given block in the chain, Ethereum has one and only one 'canonical' state, and the EVM is what defines the rules for computing a new valid state from block to block.
```
Comment (corwintines) [inline public/content/developers/docs/evm/index.md:9]:
```suggestion
The Ethereum Virtual Machine (EVM) is a core component of the Ethereum protocol, serving as a decentralized virtual environment for Ethereums state machine. It ensures that code runs consistently and securely across all nodes in the Ethereum network. The EVM's operation is powered by "[gas](/gas)".
```

Thanks @cradle0fFilth 

I agree this first section could be improved, but I don't think this changes the clarity of this section. I left a suggestion for what I would do here. Let me know what you think!

**Review by corwintines — APPROVED** [review]
Thanks @cradle0fFilth!

**Comment by corwintines** [comment]
@all-contributors can you add @cradle0fFilth for content

**Pull request merged**